### ISSUE TITLE
[Bug]: gittuf policy remote commands require a signing key

### ISSUE THREAD

**Issue by adityasaky** [gittuf/gittuf#171]
### What happened?

The remote subcommands of `gittuf policy` inherit the persistent signing key flag that's required for other policy subcommands. This flag should not be required for push and pull operations as no new metadata signatures are issued.

### Relevant log output

```Shell
$ gittuf policy remote push origin
Error: required flag(s) "signing-key" not set
Usage:
  gittuf policy remote push <remote> [flags]

Flags:
  -h, --help   help for push

Global Flags:
  -k, --signing-key string   signing key to use to sign policy file
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by adityasaky** [comment]
I was able to get around this by using a signing key (which is never actually loaded in this case), but we ought to roll out a fix for this.

**Comment by adityasaky** [comment]
Closed by #186
